brdnet/classifier.py

```diff
@@ -128,34 +135,50 @@ def load_data(args):
     val_Y: numpy.array
         The labels for val_X
```

Comment:
I had to scroll up to figure out what "val_X" was here.  This might be a stylistic preference, but I think that repeating what the variables mean is a good idea.